agent/agent.go

```diff
@@ -223,6 +225,13 @@ func (a *Agent) Start(ctx context.Context) error {
 		a.tryCleanupDirectory(a.opts.WorkingDirectory)
 	}
 
+	distro, err := a.comm.GetDistroByHostID(ctx, a.opts.HostID)
```

Comment:
Instead of a route, would it make more sense to add it as a flag in the command [here](https://github.com/ZackarySantana/evergreen/blob/96823d19a9c34e63a20bef145bc348b9a3fc5a52/operations/agent.go#L149)? I found an example of doing this [here](https://github.com/evergreen-ci/evergreen/pull/4834) 

It passes in the data [here](https://github.com/ZackarySantana/evergreen/blob/96823d19a9c34e63a20bef145bc348b9a3fc5a52/model/host/hostutil.go#L1067-L1068) and it looks like the Distro data is already loaded and ready to be passed (I haven't done testing)

Reply:
yes! did something similar to what you linked and tested that it worked